src/vulnpy/fastapi/vulnerable.py

```diff
@@ -0,0 +1,58 @@
+from fastapi import APIRouter
+from vulnpy.common import get_template
+
+from vulnpy.trigger import TRIGGER_MAP, get_trigger
+
+
+router = router = APIRouter(prefix="/vulnpy",)
+
+
+def get_root_name(name):
+    if name == "home":
+        return "/"
+    return "/{}/".format(name)
+
+
+def get_trigger_name(name, trigger):
+    return "/{}/{}/".format(name, trigger)
+
+
+def gen_root_view(name):
+    view_name = get_root_name(name)
+
+    @router.get(view_name)
+    def _view():
+        return get_template("{}.html".format(name))
```

Comment:
PYT-1705 can be resolved pretty quickly by returning an instance of `HTMLResponse`. So both here and in `_view` below, something like
```
return HTMLResponse(template)
```
the import looks like
```
from fastapi.responses import HTMLResponse
```

Reply:
oooo